wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Profile/Loader.js

```diff
@@ -1,159 +1,151 @@
     "use strict";
-    /**
-     * @constructor
-     */
-    function UiReactionProfileLoader(userID) { this.init(userID); }
-    UiReactionProfileLoader.prototype = {
+    Ajax = tslib_1.__importStar(Ajax);
+    Core = tslib_1.__importStar(Core);
+    Util_1 = tslib_1.__importDefault(Util_1);
+    Language = tslib_1.__importStar(Language);
+    class UiReactionProfileLoader {
         /**
          * Initializes a new ReactionListLoader object.
-         *
-         * @param	integer		userID
          */
-        init: function (userID) {
-            this._container = elById('likeList');
-            this._userID = userID;
+        constructor(userID) {
             this._reactionTypeID = null;
-            this._targetType = 'received';
+            this._targetType = "received";
+            this._container = document.getElementById("likeList");
+            this._userID = userID;
             this._options = {
-                parameters: []
+                parameters: {},
             };
             if (!this._userID) {
                 throw new Error("[WoltLabSuite/Core/Ui/Reaction/Profile/Loader] Invalid parameter 'userID' given.");
             }
-            var loadButtonList = elCreate('li');
-            loadButtonList.className = 'likeListMore showMore';
-            this._noMoreEntries = elCreate('small');
-            this._noMoreEntries.innerHTML = Language.get('wcf.like.reaction.noMoreEntries');
-            this._noMoreEntries.style.display = 'none';
+            const loadButtonList = document.createElement("li");
+            loadButtonList.className = "likeListMore showMore";
+            this._noMoreEntries = document.createElement("small");
+            this._noMoreEntries.innerHTML = Language.get("wcf.like.reaction.noMoreEntries");
+            this._noMoreEntries.style.display = "none";
             loadButtonList.appendChild(this._noMoreEntries);
-            this._loadButton = elCreate('button');
-            this._loadButton.className = 'small';
-            this._loadButton.innerHTML = Language.get('wcf.like.reaction.more');
-            this._loadButton.addEventListener('click', this._loadReactions.bind(this));
-            this._loadButton.style.display = 'none';
+            this._loadButton = document.createElement("button");
+            this._loadButton.className = "small";
+            this._loadButton.innerHTML = Language.get("wcf.like.reaction.more");
+            this._loadButton.addEventListener("click", () => this._loadReactions());
+            this._loadButton.style.display = "none";
             loadButtonList.appendChild(this._loadButton);
             this._container.appendChild(loadButtonList);
-            if (elBySel('#likeList > li').length === 2) {
-                this._noMoreEntries.style.display = '';
+            if (document.querySelectorAll("#likeList > li").length === 2) {
+                this._noMoreEntries.style.display = "";
```

Comment:
Was this previously a bug? Should it have been `elBySelAll`?

Reply:
Yes.